e2e_tests/tests/per_provider/normal_tests/export_public_key.rs

```diff
@@ -1,17 +1,21 @@
 // Copyright 2019 Contributors to the Parsec project.
 // SPDX-License-Identifier: Apache-2.0
+#[cfg(any(feature = "mbed-crypto-provider", feature = "cryptoauthlib-provider"))]
+use crate::per_provider::normal_tests::import_key::ECC_PUBLIC_KEY;
 use e2e_tests::TestClient;
 use parsec_client::core::interface::operations::psa_algorithm::*;
 use parsec_client::core::interface::operations::psa_key_attributes::*;
 use parsec_client::core::interface::requests::Opcode;
 use parsec_client::core::interface::requests::ResponseStatus;
 use parsec_client::core::interface::requests::Result;
+#[cfg(not(feature = "cryptoauthlib-provider"))]
 use picky_asn1_x509::RSAPublicKey;
 
+#[cfg(not(feature = "cryptoauthlib-provider"))]
 #[test]
-fn export_public_key() -> Result<()> {
+fn export_rsa_public_key() -> Result<()> {
```

Comment:
Thanks for renaming those tests 👌

Reply:
I have found this makes debugging much easier.